### ISSUE TITLE
about multi-ressolution testing

### ISSUE THREAD

**Issue by yeyujifei** [hanna-xu/DDcGAN#15]
hi, thank you for sharing the code!
I've noticed that in the muti-resolution fusion part, the resolution of the IR and VIS picture have to be fixed, which is 1:4 in your testing codes, to work out. I'm trying to change the resolution relationship by changing the multiple '4' in the following codes:
Generator.py line16
IR = deconv_ir(ir, strides = [1, 4, 4, 1], scope_name = 'deconv_ir')

Deconv.py line18-21
out = tf.nn.conv2d_transpose(out, filter = kernel, output_shape = [int(input_shape[0]), int(input_shape[1]) * 4,
		                                                                   int(input_shape[2]) * 4,
		                                                                   int(input_shape[3])],
		                             strides = strides, padding = 'SAME')

It works out for 2, 4, 8 but fails on 3, 6, 9, mainly because of the resolution difference. 

I wonder if there is a way to fuse arbitrary resolution IR and VIS pictures. Is there anything else which needs to be modified?
